src/controllers/users/approveUser.ts

```diff
@@ -8,6 +8,7 @@ import { checkExistence, constants } from '../../helpers';
 export default async (req: Request, res: Response, next: NextFunction)
 :Promise<void> => {
   const { userId } = req.params;
+  const redirectURL = `${config.server.clientURL}/`;
```

Comment:
The `/` after the clientURL is for what?
```suggestion
  const redirectURL = `${config.server.clientURL}`;
```